_env.check-docker.zsh

```diff
@@ -1,20 +1,24 @@
 #!/bin/echo Run: source
 
+declare -A container_env
 
-if [ ! -S $HOME/.rd/docker.sock ]; then 
-  echo '$HOME/.rd/docker.sock not found.'
-  echo 'Is Rancher Desktop running?'
-  return 1 
-fi
+container_env=( [Rancher]=.rd [Colima]=.colima [Docker]=.docker ) 
 
 if [ ! -S /var/run/docker.sock ]; then 
   echo '/var/run/docker.sock not found. Run:'
-  echo 'sudo ln -s $HOME/.rd/docker.sock /var/run/docker.sock'
```

Comment:
You have added some helpful checks in this PR. Maybe this file should be renamed to `_env.prerequistes.zsh` and take care of checking ALL the pre-requisites: docker. one password, STF alias definitions (both for 3.x and master)...

Reply:
Renamed